x/tokenomics/keeper/keeper_settle_pending_claims_test.go

```diff
@@ -370,3 +370,81 @@ func (s *TestSuite) TestSettlePendingClaims_DoesNotSettle_IfProofIsRequiredButMi
 func (s *TestSuite) TestSettlePendingClaims_MultipleClaimsSettle_WithMultipleApplicationsAndSuppliers() {
 	s.T().Skip("TODO_TEST: Implement that multiple claims settle at once when different sessions have overlapping applications and suppliers")
 }
+
+func (s *TestSuite) TestSettlePendingClaims_ClaimPendingAfterSettlement() {
+	// Retrieve default values
+	t := s.T()
+	ctx := s.ctx
+	sdkCtx := cosmostypes.UnwrapSDKContext(ctx)
+	sharedParams := s.keepers.SharedKeeper.GetParams(ctx)
+
+	// Set the proof parameters such that s.claim DOES NOT require a proof
+	// because the proof_request_probability is 0% and the proof_request_threshold
+	// is greater than the claims' compute units.
+	err := s.keepers.ProofKeeper.SetParams(ctx, prooftypes.Params{
+		ProofRequestProbability: 0,
+		// +1 to push the threshold above s.claim's compute units
+		ProofRequirementThreshold: s.expectedComputeUnits + 1,
+	})
+	require.NoError(t, err)
+
+	// 0. Add the claims & verify they exists
+	sessionOneClaim := s.claim
+	s.keepers.UpsertClaim(ctx, sessionOneClaim)
+
+	sessionOneStartHeight := sessionOneClaim.GetSessionHeader().GetSessionEndBlockHeight()
+	// Add a second claim with a session header corresponding to the next session.
+	sessionTwoClaim := testutilproof.BaseClaim(
+		sessionOneClaim.GetSessionHeader().GetApplicationAddress(),
+		sessionOneClaim.GetSupplierAddress(),
+		s.expectedComputeUnits,
+	)
+
+	sessionOneProofWindowCloseHeight := shared.GetProofWindowCloseHeight(&sharedParams, sessionOneStartHeight)
+	sessionTwoStartHeight := shared.GetSessionStartHeight(&sharedParams, sessionOneProofWindowCloseHeight+1)
+	sessionTwoProofWindowCloseHeight := shared.GetProofWindowCloseHeight(&sharedParams, sessionTwoStartHeight)
```

Comment:
Apologies if this was attributable to my previous suggestion(s). :sweat_smile:.

```suggestion
	sessionOneEndHeight := sessionOneClaim.GetSessionHeader().GetSessionEndBlockHeight()
	// Add a second claim with a session header corresponding to the next session.
	sessionTwoClaim := testutilproof.BaseClaim(
		sessionOneClaim.GetSessionHeader().GetApplicationAddress(),
		sessionOneClaim.GetSupplierAddress(),
		s.expectedComputeUnits,
	)

	sessionOneProofWindowCloseHeight := shared.GetProofWindowCloseHeight(&sharedParams, sessionOneEndHeight)
	sessionTwoStartHeight := shared.GetSessionStartHeight(&sharedParams, sessionOneEndHeight+1)
	sessionTwoProofWindowCloseHeight := shared.GetProofWindowCloseHeight(&sharedParams, sessionTwoStartHeight)
```